cdap-operational-stats-core/src/main/java/co/cask/cdap/operations/hdfs/AbstractHDFSStats.java

```diff
@@ -39,4 +63,85 @@
   public String getServiceName() {
     return SERVICE_NAME;
   }
+
+  public String getWebURL() {
+    try {
+      if (HAUtil.isHAEnabled(conf, getNameService())) {
+        URL haWebURL = getHAWebURL();
+        if (haWebURL != null) {
+          return haWebURL.toString();
```

Comment:
Using a non primitive type in the JMX bean classes is a lot more complicated, so we just represent it as a string